### PR TITLE
Add separate workflow tests for each OS and README badges for each

### PR DESCRIPTION
**Pull request by tbrowder**

### CODE REVIEW

**Comment by tbrowder** [comment]
All code is as before this PR except:

1. three separate workflow tests, one for each OS tested
2. the README has the test results badges added as the top line

File: .github/workflows/linux.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    tags-ignore:
+      - '*'
+  pull_request:
+
+jobs:
+  raku:
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
```
Comment (JJ) [inline .github/workflows/linux.yml:16]: Unless I'm wrong, the only difference between the two workflows is the operating system. Why don't you just add "macos" here? Even if there's a difference, you can use `if` in steps that depend on the value of the matrix that's being used.

**Review by JJ — CHANGES_REQUESTED** [review]
Besides what I've already said, are you sure this is fixing some problem? Is there any specific aspect that needs to be checked in the three operating systems? The thing is that the existing workflow, besides being simpler, it also caches dependencies, so it should take less to test anything. At any rate, no strong objection to changing to these new workflows, since there's probably a need for them, specially in Windows, but can you please create an issue to justify it and then use a single workflow that's easy to modify or evolve?

**Comment by JJ** [comment]
Another option is to leave the existing one for Ubuntu, and add another for Windows/MacOS. After all, if it's not broken, why fix it?

**Comment by tbrowder** [comment]
Nah, it isn't worth the trouble, JJ. Thanks.

**Pull request closed without merging**